### ISSUE TITLE
Maven dependency not working

### ISSUE THREAD

**Issue by cnanjo** [openEHR/archie#9]
I followed the instruction on the landing page with the following dependency (snapshot version adjusted - should be corrected on the site):

```
<dependency>
    <groupId>com.nedap.healthcare</groupId>
    <artifactId>archie</artifactId>
    <version>0.4.0-SNAPSHOT</version>
</dependency>
```

However, Maven cannot find this dependency as it cannot find a jar for the archie parent project.

However, the dependency to the child module:

```
<dependency>
            <groupId>com.nedap.healthcare.archie</groupId>
            <artifactId>bmm</artifactId>
            <version>0.4.0-SNAPSHOT</version>
</dependency>
```

is properly resolved. We might need to update the documentation.

**Comment by pieterbos** [comment]
The readme needs to be updated to reflect the new version. Also applies to some parts of the API. You can only depend on the sub modules/subprojects now. 

It’s possible to create an Archie-all module, have not done that yet.

**Comment by pieterbos** [comment]
Issue has been solved in #19 , see the new documentation in the README.md. The version is being released right now, should appear on maven central within a few hours.
